CleanProFinder/CleanProFinder.Mobile/ViewModels/ServiceProvider/ActiveRequests/ServiceProviderActiveRequestViewModel.cs

```diff
@@ -0,0 +1,83 @@
+﻿using CleanProFinder.Mobile.Services.Interfaces;
+using CleanProFinder.Mobile.Views.Info;
+using CleanProFinder.Mobile.Views.ServiceProvider.ActiveRequests;
+using CleanProFinder.Mobile.ViewModels.Info;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using CleanProFinder.Shared.Dto.Requests;
+
+namespace CleanProFinder.Mobile.ViewModels.ServiceProvider.ActiveRequests;
+
+public partial class ServiceProviderActiveRequestViewModel : ObservableObject, IQueryAttributable
+{
+    private readonly IDialogService _dialogService;
+    private readonly IRequestService _requestService;
+
+    public ServiceProviderActiveRequestViewModel(IDialogService dialogService, IRequestService requestService)
+    {
+        _dialogService = dialogService;
+        _requestService = requestService;
+    }
+
+    [ObservableProperty]
+    private Guid _requestId;
+
+    [ObservableProperty]
+    private RequestFullInfoProviderViewDto _request;
+
+    public float Price { get; set; }
```

Comment:
Better to make it an ObservableProperty

Reply:
Price is not used on the ServiceProviderActiveRequestPage